std/getopt.d

```diff
@@ -1106,7 +1106,7 @@ private struct configuration
 private bool optMatch(string arg, scope string optPattern, ref string value,
     configuration cfg) @safe
 {
-    import std.array : split;
+    import std.algorithm : splitter;
```

Comment:
```
grep -nr 'import std.algorithm : ' $(find etc std -name '*.d') ; test $? -eq 1
--
&nbsp; | std/getopt.d:1109:    import std.algorithm : splitter;
```

I think you need to `import std.algorithm.mutation : splitter;` or whatever the submodule is.